src/org/usfirst/frc/team2335/robot/Robot.java

```diff
@@ -73,8 +88,28 @@
 	@Override
 	public void robotInit() //Runs once to initialize all global variables
 	{
-		//TODO: Possibly move the camera init to its own private function,
-		//		since it's longer and more complex.		
+		
+		driveTrain = new DriveTrain();
+		climb = new Climb();
+		shooter = new Shooter();
+		ultraSound = new Ultrasound();
+		vision = new Vision();
+		
+		//This one comes last or else your code dies just like you if you don't define it last
```

Comment:
While technically true, that isn't the true point here. In particular, we should keep the codebase professional so that no one gets distracted by extraneous commentary down the line.